best-practices.md

```diff
@@ -253,10 +253,10 @@ STAC version. Otherwise some behaviour of functionality may be unpredictable (e.
 
 ## Use of links
 
-The main catalog specification allows both relative and absolute links, and says that `self` links are not required, but are 
+The main STAC specifications allow both relative and absolute links, and says that `self` links are not required, but are 
```

Comment:
Maybe `The main STAC specifications allow` -> `The STAC specification allows`

What is meant by `main`?  Are there other specifications in play here?

Reply:
I'll work to clarify this. Catalog, Item and Collection are all technically their own 'spec', designed to potentially use stand alone, so that's all I meant. I was changing it to not seem like it was implying just the 'catalog' spec. But I see how 'main' is confusing.